change_log/next/FE-2175-grouped-character-uncontrolled-support.yml

```diff
@@ -0,0 +1,2 @@
+Bug Fixes: "Fixed issue where component did not supporter uncontrolled use. (Components: GroupedCharacter)"
```

Comment:
Bug Fixes: "Fixed issue where component did not support uncontrolled use. (Components: GroupedCharacter)"